cg_hermes/constants/tags.py

```diff
@@ -73,6 +74,12 @@ class VariantTags(StrEnum):
     VCF2CYTOSURE: str = "vcf2cytosure"
     VCF: str = "vcf"
     VCF_INDEX: str = "vcf-index"
+    VCF_MT: str = "vcf_mt"
+    VCF_MT_INDEX: str = "vcf_mt_index"
+    VCF_MT_CLINICAL: str = "clinical_mt"
+    VCF_MT_CLINICAL_INDEX: str = "clinical_mt_index"
+    VCF_MT_RESEARCH: str = "research_mt"
+    VCF_MT_RESEARCH_INDEX: str = "research_mt_index"
```

Comment:
I would split this to the following tags, all of them already would exist:
vcf_mt: vcf, mitochondria
vcf_mt_index: vcf-index, mitochondria
clinical_mt: vcf, clinical, mitochondria
clinical_mt_index: vcf-index, clinical, mitochondria
research_mt:  vcf, research, mitochondria
research_mt_index:  vcf-index, research, mitochondria